Resources/Info.plist

```diff
@@ -42,16 +42,10 @@
 	<string></string>
 	<key>NSCameraUsageDescription</key>
 	<string>Frames from the camera are passed to ML models to get semantic segmentation, object detection and extract other features</string>
-	<key>NSLocationAlwaysAndWhenInUseUsageDescription</key>
-	<string>Location and heading information is used for position estimation, localization and AR-related tasks</string>
-	<key>NSLocationAlwaysUsageDescription</key>
-	<string>Location and heading information is used for position estimation, localization and AR-related tasks</string>
 	<key>NSLocationWhenInUseUsageDescription</key>
 	<string>Location and heading information is used for position estimation, localization and AR-related tasks</string>
 	<key>UIBackgroundModes</key>
```

Comment:
Since there's no background modes we're using, we may remove this key completely.